pkg/generators/forwarding/fluentd/generators.go

```diff
@@ -0,0 +1,210 @@
+package fluentd
+
+import (
+	"bytes"
+	"fmt"
+	"sort"
+	"strings"
+	"text/template"
+
+	logforward "github.com/openshift/cluster-logging-operator/pkg/apis/logging/v1alpha1"
+	"github.com/openshift/cluster-logging-operator/pkg/generators"
+	"github.com/openshift/cluster-logging-operator/pkg/logger"
+	"k8s.io/apimachinery/pkg/util/sets"
+)
+
+var (
+	defaultInfraProjectPatterns = []string{"default", "openshift", "openshift-", "kube-"}
```

Comment:
we're calling them `default` here but we have no means to change it... can we just drop 'default'?

Reply:
removed as not need